qiskit/transpiler/passes/optimization/optimize_1q_commutation.py

```diff
@@ -10,7 +10,7 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Reduce 1Q gate complexity by commuting through 2Q gates and resynthesizing."""
+"""Reduce 1Q gate complexity by commuting through 2Q gates and re-synthesizing."""
```

Comment:
Using the hyphen in constructed words like this (and multiqubit, etc) is I think falling out of fashion a bit, especially in US English.  It's a bit more common to put one in still in British English, I think, but generally it's fine without it.

Reply:
Thanks, addressing it in 4f77873bf3cb68836a90f9a8661758aaf5a64ede